File: src/main/java/com/back/mozu/domain/setting/dto/SettingUpdateResponseDto.java
package com.back.mozu.domain.setting.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.time.LocalTime;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class SettingUpdateResponseDto {
    private Integer totalTables;
    private LocalTime openingTime;
    private LocalTime closingTime;
}

File: src/main/java/com/back/mozu/domain/setting/dto/SettingUpdateResponseDto.kt
package com.back.mozu.domain.setting.dto


import java.time.LocalTime

data class SettingUpdateResponseDto (
    val totalTables: Int,   // ResponseDTO는 불변, 응답값이 항상 있어야 함
    val openingTime: LocalTime,
    val closingTime: LocalTime
)